services/cinema_hall.py

```diff
@@ -0,0 +1,19 @@
+from django.db.models import QuerySet
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:

```

Comment:
Indicate in the return type annotation a QuerySet of what instances you expect to return here and in the other classes functions as well